common/restore.py

```diff
@@ -64,5 +62,5 @@ def restore_from_dump(
     )
     restore_from_dump(
         "opt/backup.dump",
-        os.environ.get('PGDATABASE')
+        os.environ.get('PGDATABASE') if os.environ.get('PGDATABASE') else ""
```

Comment:
```
os.environ.get('PGDATABASE', "")
```

Reply:
Thanks for the comment. Actually this environment variable (`PGDATABASE`) is used for Postgres only and I wanted to make this script reusable for Mongo as well. I did `os.environ.get('PGDATABASE', '')` for now.

Reply:
By the way for Mongo, I added the `--db`/`--nsInclude` option in the suggested commands.

Reply:
See my comment below.